lib/foreman_tasks_core/task_launcher/group.rb

```diff
@@ -0,0 +1,122 @@
+require 'foreman_tasks_core/runner'
+
+module ForemanTasksCore
+  module TaskLauncher
+    class AbstractGroup < Batch
+      class GroupParentAction < ForemanTasksCore::TaskLauncher::ParentAction
+        def plan(launcher, input_hash)
+          launcher.launch_children(self, input_hash)
+          sequence do
+            results = plan_self
+            plan_action BatchCallback, launcher.prepare_batch(input_hash), results.output[:results]
```

Comment:
As suggested in https://github.com/theforeman/foreman-tasks/pull/386/files#r267295169, the responsiblity for formating the callback input might be better in this action. Something like:

```ruby
prepare_callback_input(input_hash, launcher.planned_task_ids)
# ...
def prepare_callback_input(input_hash, task_ids)
  task_ids.reduce({}) do |hash, task_id|
    hash.update(task_id => input_hash[task_id]['action_input']['callback'])
  end
end

```